terraform/identitysandbox.gov.tf

```diff
@@ -1,2 +0,0 @@
-# login.gov DNS configuration is managed separately:
-# https://github.com/18F/identity-devops/blob/master/doc/structure-of-this-repository.md#terraform-dns-directory
```

Comment:
Can we leave these in? Helps to point folks to the right place. We can mention it's a private repo, if that's the issue.

https://github.com/18F/identity-devops/blob/main/doc/structure-of-this-repository.md#terraformdns-directory

Reply:
Ah, thanks. Yes, I was removing these because it was a broken link. I'll update it.